<ui version="4.0" >
 <class>astSource</class>
 <widget class="QWidget" name="astSource" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QRadioButton" name="rbDrive" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>DVD Drive</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="cbDrives" />
      </item>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="rbFolder" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Folder</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="KUrlRequester" name="urFolder" />
      </item>
      <item row="2" column="0" >
       <widget class="QRadioButton" name="rbIso" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>ISO Image</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="KUrlRequester" name="urIso" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>157</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>rbDrive</sender>
   <signal>toggled(bool)</signal>
   <receiver>astSource</receiver>
   <slot>rbDriveToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>40</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel" >
     <x>0</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbFolder</sender>
   <signal>toggled(bool)</signal>
   <receiver>astSource</receiver>
   <slot>rbFolderToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>55</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel" >
     <x>1</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbIso</sender>
   <signal>toggled(bool)</signal>
   <receiver>astSource</receiver>
   <slot>rbIsoToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>82</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel" >
     <x>0</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>rbDriveToggled(bool)</slot>
  <slot>rbFolderToggled(bool)</slot>
  <slot>rbIsoToggled(bool)</slot>
 </slots>
</ui>
